src/slider/slider.component.spec.ts

```diff
@@ -0,0 +1,138 @@
+import { Component } from "@angular/core";
+import { TestBed } from "@angular/core/testing";
+import { By	} from "@angular/platform-browser";
+
+import { Slider } from "./slider.component";
+import { CommonModule } from "@angular/common";
+
+@Component({
+	template: `
+	<ibm-slider
+		[shiftMultiplier]="shiftMultiplier"
+		(valueChange)="onValueChange()"
+		[disabled]="disabled"
+		[max]="max"
+		[min]="min">
+	</ibm-slider>`
+})
+class SliderTest {
+	disabled = false;
+	max = 100;
+	min = 0;
+	shiftMultiplier = 28;
+	onValueChange() {}
+}
+
+describe("Dropdown", () => {
+	let fixture, element, wrapper;
+	beforeEach(() => {
+		TestBed.configureTestingModule({
+			declarations: [
+				Slider,
+				SliderTest
+			],
+			imports: [
+				CommonModule
+			]
+		});
+	});
+
+	beforeEach(() => {
+		fixture = TestBed.createComponent(SliderTest);
+		element = fixture.debugElement.query(By.css("ibm-slider"));
+		wrapper = fixture.componentInstance;
+		fixture.detectChanges();
+	});
```

Comment:
Some of the tests assign the fixture, element, wrapper within the` it()` blocks and some are using a second  `beforeEach()`.
I favour this way, as it reduces lots of duplicate code, but maybe we can commit to one way or another and update the tests for consistency and ease of contribution in the future